fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/Coalesce.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.expression;
+
+import java.util.List;
+
+/**
+ * Represents the SQL COALESCE function.
+ */
+public class Coalesce extends VarArgFunction {
```

Comment:
there is no call hierarchy for this class.   Is this needed?  maybe leave an `implNote` to indicate the purpose

Reply:
it's old code from an earlier implementation. There's a whole set of this predicate stuff which can probably be cut out now. But I'm not risking that right now.

Reply:
I  think we should add a backlog item to remove this one. https://github.com/IBM/FHIR/issues/2317